src/Sarif.Multitool.FunctionalTests/TestData/UriMustBeAbsolute/Invalid.sarif

```diff
@@ -1,27 +1,18 @@
 {
-  "$schema": "json.schemastore.org/sarif-2.0.0",
-  "version": "2.0.0",
+  "$schema": "json.schemastore.org/sarif-2.0.0-csd.2.beta.2019-01-24",
+  "version": "2.0.0-csd.2.beta.2019-01-24",
   "runs": [
     {
       "tool": {
         "name": "CodeScanner",
-        "downloadUri": "tools/codescanner/download.html"
-      },
-      "results": [
-        {
-          "message":  { "text":  "Some testing occurred."},
-          "ruleId": "TST0001",
-          "level": "error",
-          "workItemUris": [
-            "example.com/my-project/issues/42"
-          ]
-        }
-      ],
-      "originalUriBaseIds": {
-        "SRCROOT": { "uri": "Code/sarif-sdk/src" }
-      },
-      "resources": {
-        "rules": [
+        "downloadUri": "tools/codescanner/download.html",
+        "notificationsMetadata": [
```

Comment:
>notificationsMetadata [](start = 9, length = 21)

:-) this doesn't actually expose the copy/paste bug I found, which would have been triggered if the number of notification metadatas was different from the number of rule metadatas.

Reply:
this change actually did provoke a test failure, i developed the test first to prove it, then took a fix. I've added a second notification, however. I also corrected the notification HTML page name to correspond to the id. :) This change so close to perfection, let's just nail it. 

---
In reply to: [255316074](https://github.com/Microsoft/sarif-sdk/pull/1259#discussion_r255316074) [](ancestors = 255316074)

Reply:
remember, these tests are driven off JSON paths to the issue location. so after my original change, the test failed because we found an issue at "runs[0].tool.notificationsMetadata[0].helpUri". which wasn't documented in the expected results properties.

---
In reply to: [255346258](https://github.com/Microsoft/sarif-sdk/pull/1259#discussion_r255346258) [](ancestors = 255346258,255316074)